DOCSP-50554: Add extra AWS info to OIDC page (#648)
mongodb/docs-csharp: source/fundamentals/authentication/oidc.txt

            .CreateOidcCredential("k8s")
         var client = new MongoClient(mongoClientSettings);

> If your application runs on `Amazon Elastic Kubernetes Service <https://aws.amazon.com/eks/>`__
> (EKS), you will need to create an ``AWS_WEB_IDENTITY_TOKEN_FILE``
> environment variable that points to a file containing your OIDC token in addition to
> the preceding Kubernetes OIDC authentication steps. To learn more about authenticating to
> MongoDB from AWS applications, see the :ref:`csharp-authentication-aws` guide.

Reviewer: ```suggestion
If your application runs on `Amazon Elastic Kubernetes Service <https://aws.amazon.com/eks/>`__
(EKS), you must create an ``AWS_WEB_IDENTITY_TOKEN_FILE``
environment variable that points to the file containing your OIDC token, in addition to
completing the preceding Kubernetes OIDC authentication steps. To learn more about authenticating to
MongoDB from AWS applications, see the :ref:`csharp-authentication-aws` guide.
```
